tests/test_wsi_registration.py

```diff
@@ -219,3 +219,22 @@ def test_match_histograms():
     norm_image_a, norm_image_b = match_histograms(image_a, image_b)
     assert np.all(norm_image_a == expected_output)
     assert np.all(norm_image_b == image_b)
+
+
+def test_register(fixed_image, moving_image, fixed_mask, moving_mask):
+    fixed_img = imread(fixed_image)
+    moving_img = imread(moving_image)
+    fixed_msk = imread(fixed_mask)
+    moving_msk = imread(moving_mask)
+
+    pre_transform = np.array([[-1, 0, 337.8], [0, -1, 767.7], [0, 0, 1]])
+    moving_img = cv2.warpAffine(
+        moving_img, pre_transform[0:-1][:], fixed_img.shape[:2][::-1]
+    )
+    moving_msk = cv2.warpAffine(
+        moving_msk, pre_transform[0:-1][:], fixed_img.shape[:2][::-1]
+    )
+
+    df = DFBRegister()
+    _ = df.register(fixed_img, moving_img, fixed_msk, moving_msk)
```

Comment:
This needs to return an image, transformation or a metric which should be compared against GT.

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/7be8b4c7387ead7ee0cf082f14ded5ab87bc771a/tests/test_wsi_registration.py#L238-L244